src/containers/MyNdla/Arena/components/ArenaCategoryForm.tsx

```diff
@@ -158,20 +149,20 @@ const ArenaCategoryForm = ({
           required: false,
         }}
         render={({ field, fieldState }) => (
-          <FormControl id="editor" isRequired isInvalid={!!fieldState.error?.message}>
-            <StyledLabel textStyle="label-small">{t("myNdla.arena.admin.category.form.description")}</StyledLabel>
+          <FieldRoot required invalid={!!fieldState.error?.message}>
+            <FieldLabel>{t("myNdla.arena.admin.category.form.description")}</FieldLabel>
             <FieldErrorMessage>{fieldState.error?.message}</FieldErrorMessage>
-            <StyledInput {...field} />
-          </FormControl>
+            <FieldInput {...field} />
+          </FieldRoot>
         )}
       />
       <Controller
         control={control}
         name={"parentCategoryId"}
         rules={{ required: false }}
         render={({ field, fieldState }) => (
-          <FormControl id="parentCategoryId" isInvalid={!!fieldState.error?.message}>
-            <StyledLabel textStyle="label-small">{t("myNdla.arena.admin.category.form.parentCategoryId")}</StyledLabel>
+          <FieldRoot invalid={!!fieldState.error?.message}>
```

Comment:
Her brekker du koblingen mellom label og select. Du kan wrappe Select med en `<Field.Select asChild>`. Vi eksporterer den ikke, men ark har den!

Reply:
Det var ikke meningen å få med de endringene for checkbox/select 😅 reverterer det

Reply:
Var ikke meningen å ta med de endringene egentlig 😅 revertert nå!